src/debugger/server/Stepper.js

```diff
@@ -32,8 +31,6 @@ export class Stepper {
   // The same node in two different excutions contexts (e.g. recursive call)
   // will not be detected. Check the stackSize (via realm) in those cases.
   isAstLocationChanged(ast: BabelNode) {
-    // we should only step to statements
-    if (!IsStatement(ast)) return false;
```

Comment:
Removing because the purpose of the `IsStatement` check is to only step once per line. This is taken care of with the change in `checkAndUpdateLastExecuted` that ignores columns.